nil/internal/execution/testaide.go

```diff
@@ -35,15 +96,16 @@ func GenerateZeroState(t *testing.T, ctx context.Context,
 	require.NoError(t, err)
 	defer g.Rollback()
 
-	zerostateCfg, err := ParseZeroStateConfig(DefaultZeroStateConfig)
+	zerostateCfg := DefaultZeroStateConfig
```

Comment:
Shouldn't we copy this value?

Reply:
Changed to `zerostateCfg, err := CreateDefaultZeroStateConfig(MainPublicKey)`